subprojects/core/src/main/java/org/gradle/internal/service/scopes/BuildSessionScopeServices.java

```diff
@@ -167,6 +169,16 @@ BuildOperationExecutor createBuildOperationExecutor(
         );
     }
 
+    LoggingBuildOperationNotificationBridge createLoggingBuildOperationNotificationBridge(LoggingManagerInternal loggingManagerInternal, ListenerManager listenerManager) {
```

Comment:
We can't have this be build session scoped. Two build sessions can happen concurrently (only with GradleBuild task), but logging system is JVM global. We will have to move this up to global scope services.